Code/Framework/AzCore/AzCore/Serialization/SerializeContext.h

```diff
@@ -1938,7 +1938,7 @@ namespace AZ
         if (m_context->IsRemovingReflection())
         {
             // Delete any attributes allocated for this call.
-            for (auto attributePair : attributes)
+            for (auto &attributePair : attributes)
```

Comment:
coding standards: ampersand should be on the auto

```suggestion
            for (auto& attributePair : attributes)
```

Also most of these range based for loops don't need to change.
They are iterating over pointers.

Reply:
Well, this specific one is iterating over pairs that contain the pointer :)

Reply:
You are right, my mistake.